packages/foundry/test/vault/integration/aura/AuraAdapter.t.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+
+import { AuraAdapter, SafeERC20, IERC20, IERC20Metadata, Math, IAuraBooster, IAuraRewards, IAuraStaking, IStrategy, IAdapter, IWithRewards } from "../../../../src/vault/adapter/aura/AuraAdapter.sol";
+import { AuraTestConfigStorage, AuraTestConfig } from "./AuraTestConfigStorage.sol";
+import { AbstractAdapterTest, ITestConfigStorage } from "../abstract/AbstractAdapterTest.sol";
+import { MockStrategyClaimer } from "../../../utils/mocks/MockStrategyClaimer.sol";
+
```

Comment:
Could you add `test__claim` ?

Reply:
Sure, I already had it passing but I deleted the override because I figured the abstract would already handle it